nni/runtime/tuner_command_channel/channel.py

```diff
@@ -48,6 +50,19 @@ def disconnect(self) -> None:
     #     ...
     # def receive(self) -> Command | None:
     #     ...
+    def send(self, command: BaseCommand) -> None:
+        command_json = command.dump()
+        self._channel.send(command_json)
```

Comment:
Please add a temporary compatibility layer. For example `self._channel.send(command._to_legacy_command_type(), command_json)`.